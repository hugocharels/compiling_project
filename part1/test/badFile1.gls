!! I am a well written file  !!

LET Kind_file BE
  IN(a):
  a = 
  OUT(a):
END