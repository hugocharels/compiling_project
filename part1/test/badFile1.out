token: LET              lexical unit: LET
token: Kind_file        lexical unit: PROGNAME
token: BE               lexical unit: BE
token: IN               lexical unit: INPUT
token: (                lexical unit: LPAREN
token: a                lexical unit: VARNAME
token: )                lexical unit: RPAREN
token: :                lexical unit: COLUMN
token: a                lexical unit: VARNAME
token: =                lexical unit: ASSIGN
